ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ContractIdIT.scala

```diff
@@ -26,171 +27,202 @@ import scala.util.{Failure, Success, Try}
 // - Central committer ledger implementations (sandboxes, KV...) may accept non-suffixed CID
 // - Distributed ledger implementations (e.g. Canton) must reject non-suffixed CID
 final class ContractIdIT extends LedgerTestSuite {
-  private[this] val nonSuffixedV1Cid = (0 to 32).map("%02x".format(_)).mkString
-  private[this] val suffixedV1Cid = (0 to 48).map("%02x".format(_)).mkString
 
-  private[this] def camlCase(s: String) =
-    s.split("[ -]").iterator.map(_.capitalize).mkString("")
-
-  List[(String, String, Boolean, Features => Boolean, String)](
-    (
-      nonSuffixedV1Cid,
-      "non-suffixed V1",
-      true,
-      features => features.contractIds.v1.isNonSuffixed || features.contractIds.v1.isBoth,
-      "non-suffixed V1 contract IDs are not supported",
+  List(
+    TestConfiguration(
+      description = "v0",
+      example = v0Cid,
+      accepted = true,
+      isSupported = features => features.contractIds.v0.isSupported,
```

Comment:
```suggestion
      isSupported = features => false,
```

Reply:
So we never want to run this test case?

Reply:
We do not have any ledger that support V0. 
Sandbox classic was the only one, and it is gone.

Reply:
sandbox classic and sandbox on x are both still in the repo and support it. Let’s not change that in this PR and instead separately drop support.